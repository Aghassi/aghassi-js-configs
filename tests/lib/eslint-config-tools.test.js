const eslintTools = require('../../lib/eslint-config-tools');
const validateConfig = require('./__helpers__/configSnapshot');

// Use the helper suite to generate a test suite to run
validateConfig('eslint-config-tools', eslintTools);
