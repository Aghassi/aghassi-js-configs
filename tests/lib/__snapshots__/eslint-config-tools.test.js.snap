// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`eslint-config-tools config matches default config snapshot 1`] = `
Object {
  "extends": Array [
    "<PROJECT_ROOT>/node_modules/eslint-config-airbnb-base/index.js",
    "<PROJECT_ROOT>/node_modules/eslint-config-prettier/index.js",
    "<PROJECT_ROOT>/node_modules/eslint-plugin-eslint-comments/lib/configs/recommended.js",
  ],
  "plugins": Array [
    "prettier",
  ],
  "rules": Object {
    "prettier/prettier": Array [
      "error",
    ],
  },
}
`;
